package.json

```diff
@@ -31,6 +31,7 @@
     "babel-loader": "^8.0.6",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
+    "bootstrap": "^4.3.1",
```

Comment:
is this supposed to be here?

Reply:
yes for the basic bootstrap css that Reactstrap doesn't provide